README.md

```diff
@@ -1,77 +1,45 @@
 [![License: AGPL v3](https://img.shields.io/badge/License-AGPL_v3-green.svg)](https://www.gnu.org/licenses/agpl-3.0)
 
+# ArmoniK.Core
 
-# Project components
+## What is ArmoniK.Core?
 
-This page intends to describe the different software projetcs and components used for ArmoniK 
-internals. To learn how to install or use ArmoniK, please see these repositories:
-  * install: 
-  * use: 
+This project is part of the [ArmoniK](https://github.com/aneoconsulting/ArmoniK) project. ArmoniK.Core is responsible for the implementation of the services needed for ArmoniK which are defined in [ArmoniK.Api](https://github.com/aneoconsulting/ArmoniK.Api).
 
+ArmoniK.Core provides services for submitting computational tasks, keeping track of the status of the tasks and retrieving the results of the computations. The tasks are processed by external workers whose interfaces are also defined in ArmoniK.Api. ArmoniK.Core sends tasks to the workers, manages eventual errors during the execution of the tasks and the storage of the task's results.
 
-## Common Library
+More detailed information on the inner working of ArmoniK.Core is available [here](https://aneoconsulting.github.io/ArmoniK.Core/).
 
-**ArmoniK.Common** provides the components required by all the other compononents of ArmoniK.
+## Development Setup
 
-## Control
+ArmoniK.Core can be setup only on Linux machines. For Windows users, it is possible to do it on [WSL2](https://learn.microsoft.com/en-us/windows/wsl/about).
 
-Different projects are available in the control plan:
+### Prerequisites
 
-* **ArmoniK.Control.Submitter** is a web server providing the gRPC services required by the 
-SubmitterService client from SDK. It provides the following services:
-  * Session creation
-  * Task submission
-  * Results retrieving
-  * Task cancellation
-  * Session cancellation
+- [Terraform](https://www.terraform.io/) version >= 1.4.2
+- [Just](https://github.com/casey/just) >= 1.8.0
+- [Dotnet](https://dotnet.microsoft.com/en-us/) >= 6.0
+- [Docker](https://www.docker.com/) >= 20.10.16
+- [GitHub CLI](https://cli.github.com/) >= 2.23.0
```

Comment:
GitHub CLI  should be flagged as optional,  its not necessary to install and deploy the code but may simplify the development